packages/issuer-sdk-js/src/human.ts

```diff
@@ -42,14 +46,14 @@ async function upsertWallet(
 
 export async function createHuman(
   config: CreateIssuerConfig,
-  current_public_key: string,
-  wallet: Omit<UpsertWalletReqParams, "id" | "human_id">,
+  human: CreateProfileReqParams,
+  wallet: Omit<UpsertWalletReqParams, "human_id">,
 ) {
-  const human_id = crypto.randomUUID();
-  const wallet_id = crypto.randomUUID();
+  const human_id = human.id ?? crypto.randomUUID();
+  const wallet_id = wallet.id ?? crypto.randomUUID();
 
```

Comment:
how about another method that can also be used on createActionInput. we can call it sth like ensureId
```
const ensureId = (id?:string) => id || crypto.randomUUID() 
// use it else where
const human_id = ensureId(human.id)
 const wallet_id = ensureId(wallet.id)
const payload = { ...params, id: ensureId(params.id)};
```